microsoft-iis-mixin/dashboards/microsoft-iis-overview.libsonnet

```
@@ -0,0 +1,1273 @@
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      min: 0,
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'percent',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+local outputCacheHitRatioPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'increase(windows_iis_server_output_cache_hits_total{job=~"$job", instance=~"$instance"}[$__interval:]) / clamp_min(increase(windows_iis_server_output_cache_queries_total{job=~"$job", instance=~"$instance"}[$__interval:]), 1) * 100',
+      datasource=promDatasource,
+      legendFormat='{{job}} - {{instance}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Output cache hit ratio',
+  description: 'The current output cache hit ratio for an IIS site.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'percent',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+{
+  grafanaDashboards+:: {
+    'microsoft-iis-overview.json':
+      dashboard.new(
+        'Microsoft IIS overview',
+        time_from='%s' % $._config.dashboardPeriod,
+        tags=($._config.dashboardTags),
+        timezone='%s' % $._config.dashboardTimezone,
+        refresh='%s' % $._config.dashboardRefresh,
+        description='',
+        uid=dashboardUid,
+      )
+
```

dashboard-to-mixin *doesn't* add links to other dashboards, need to manually do this afterwards:

```suggestion
      .addLink(grafana.link.dashboards(
        asDropdown=false,
        title='Other Microsoft IIS dashboards',
        includeVars=true,
        keepTime=true,
        tags=($._config.dashboardTags),
      ))
```